xspec/xspec
# Fix issue 204 (xspec.sh fails to generate code coverage report when stderr contains other outputs)

This pull request derives from #342. So needs to be handled after that.

---

Fixes #204.

#### How to test this pull request

1. Check out 3592952301d3bd5356489ceb4ccdf1f971a65ba5 which reproduces the problem.
1. Go to `[XSpec]/test`.
1. Do `export SAXON_JAR=/path/to/saxon9ee.jar`. (Saxon-PE should be fine too, but I haven't tested.)
1. Run `./run-bats.sh --tap`.
    ==> Test 14 should fail.
1. Check out 524d5046b27a9099008db7cc24ffea09d8c269c5 which fixes the problem in the same way as [`xspec.bat`](https://github.com/xspec/xspec/blob/0e21fca458505ad9b2048e3b42a06a99952a7e3e/bin/xspec.bat#L76-L89).
1. Run `./run-bats.sh --tap`.
    ==> Test 14 should pass.
